Reference/google/protobuf/unittest_mset.pb.swift

```diff
@@ -304,7 +304,7 @@ struct ProtobufUnittest_RawMessageSet: SwiftProtobuf.Message {
 
 extension Proto2WireformatUnittest_TestMessageSet {
   var ProtobufUnittest_TestMessageSetExtension1_messageSetExtension: ProtobufUnittest_TestMessageSetExtension1 {
-    get {return getExtensionValue(ext: ProtobufUnittest_TestMessageSetExtension1.Extensions.message_set_extension)}
+    get {return getExtensionValue(ext: ProtobufUnittest_TestMessageSetExtension1.Extensions.message_set_extension) ?? ProtobufUnittest_TestMessageSetExtension1()}
```

Comment:
@tbkka this PR ok?  The defaultInstance thing would be independent, and is tracked in that other issue since it seems like extensions and fields should be common on this.